### PR TITLE
Adding papers on Domain adaptation and prior knowledge integration

### PR DESCRIPTION
**Pull request by lvapeab**
Hi!
Papers on prior knowledge integration belong to the interactive-predictive framework for (N)MT. If you consider them adequate, maybe it is worth adding a new category.
Cheers!

### CODE REVIEW

**Comment by lvapeab** [comment]
Thanks for the interest!

I would rephrase it saying that interactive NMT refers to a different _decoding_ strategy, that involves partial supervision from users. It can be seen as an alternative framework to translation post-editing (or other MT output correction strategy).

Indeed, this is tightly related with the scenarios you describe. Since we have this partial feedback, we can improve our models as a bandit learning/reinforcement learning setup. In fact, [this paper](https://arxiv.org/abs/1805.01553) studies this scenario (I missed it in my PR and might be convenient to include it). Another option for learning within the interactive MT framework is to learn from corrected sentences (obtained either via post-editing or interactive MT). There are also a bunch of works regarding this topic. 

It is somewhat hard to frame the interactive framework into a single category, because it relates several topics and the frontiers are diffuse. Maybe these works can be categorized into a new subsection like _Human-centered NMT_? What do you think?

Cheers!

**Comment by minicheshire** [comment]
Got it. The idea of *Human-centered NMT* is brilliant! 

We can say that interactive NMT, as well as the further improvements via bandit learning/reinforcement learning, integrates user feedback into the current NMT architecture and it's still in an end-to-end manner. Automatic Post-Editing, on the other hand, is a two-stage approach which treats translating and improving by feedback as two separate tasks. 

The *human-centered* idea is what interactive NMT and automatic post-editing share in common. I will merge the pr and re-organize the related papers later.

Thanks again for your great advice!

**Comment by lvapeab** [comment]
A small clarification: I don't think automatic post-editing belongs to human-centered MT. The idea of APE is to automatically correct the outputs of a MT system, but (in principle) there's no human intervention in this scenario.

I canalso collect some papers on human-centered SMT, if you think is valuable to add it on the SMT secton :)

**Comment by minicheshire** [comment]
Hi! 

In my opinion, automatic post-editing separate the step of using human feedback from the (pure) machine translation step. The idea of APE is indeed to automatically correct the outputs of MT, but I would say that the correcting step also involves human intervention -- in the form of the provided data --- as there are golden post-edited sentences paralleled raw translation sentences. Compared with the canonical MT architecture where there are only two types of data provided, I tend to insist on classifying APE into the session of human-centered MT. 

By the way, you are welcomed to contribute on human-centered SMT!

Cheers!
